catalan.ini

```
@@ -0,0 +1,670 @@
+MonsterCounter = "Comptador de Metroids"
+MonsterCounter_Tip = "Canvia el tipus de Comptador en l'indicador en pantalla"
+MonsterCounter_Disabled_Tip = "No mostrar el Comptador"
+MonsterCounter_Local = "Local"
+MonsterCounter_Local_Tip = "Mostra els Metroids restants en l'àrea actual"
+MonsterCounter_Global = "Global"
+MonsterCounter_Global_Tip = "Mostra els Metroids restants en el planeta"
+Title_Notifications = "Notificacions"
+Hints = "Consells del joc"
+Hints_Tip = "Mostra els consells per a principiants durant la partida"
+ScanNotifications = "Notificació d'Escaneig"
+ScanNotifications_Tip = "Mostra un missatge en pantalla durant Esdeveniments d'Escaneig"
+MedalNotifications = "Notificació de Medalles"
+MedalNotifications_Tip = "Mostra un missatge en pantalla quan es desbloquegen Medalles"
+Language = "Idioma"
+Language_Tip = "Canvia l'idioma del text del joc"
+Author = "Traductors:"
+
+[OptionsSound]
+Title_Sound = "Opcions de So"
+SoundVolume = "Volum dels Efectes de So"
+SoundVolume_Tip = "Volum principal per als Efectes de So"
+MusicVolume = "Volum de la Música"
+MusicVolume_Tip = "Volum principal per a la Música Ambiental"
+
+[OptionsControl]
+Title_Control = "Opcions de Control"
+AimingStyle = "Estil d'Apuntat"
+AimingStyle_Tip = "Alterna entre l'estil d'apuntat de Zero Mission i Super Metroid"
+AimingStyle_1Button = "1 Botó"
+AimingStyle_1Button_Tip = "Prem [Amunt] o [Avall] mentre apuntes per a elegir la direcció"
+AimingStyle_2Buttons = "2 Botons"
+AimingStyle_2Buttons_Tip = "Apunta diagonalment usant 2 botons diferents"
+MissileStyle = "Estil de Selecció de Míssil"
+MissileStyle_Tip = "Alterna entre la selecció d'armes de Zero Mission i Super Metroid"
+MissileStyle_Toggle = "Alternar"
+MissileStyle_Toggle_Tip = "Els Míssils romanen activats, prem de nou per a desactivar"
+MissileStyle_Hold = "Mantenir"
+MissileStyle_Hold_Tip = "Els Míssils estaran activats mentre es mantingui el botó"
+ClassicMorphball = "Morfosfera Clàssica"
+ClassicMorphball_Tip = "Activació / Desactivació Clàssica de Morfosfera"
+ClassicMorphball_Enabled_Tip = "Transformar-se amb les comandes clàssiques i amb el botó [Morfosfera]"
+ClassicMorphball_Disabled_Tip = "Prem [Morfosfera] per a Activar / Desactivar la Morfosfera"
+SpiderballActivation = "Mode de Aracnosfera"
+SpiderballActivation_Tip = "Activació / Desactivació clàssica o nova de la Aracnosfera"
+SpiderballActivation_Toggle = "Nova (alternada)"
+SpiderballActivation_Toggle_Tip = "Prem [Apuntar] per a Activar / Desactivar"
+SpiderballActivation_Classic = "Clàssica"
+SpiderballActivation_Classic_Tip = "[A baix] en mode Morfosfera per a activar, [Salt] per a desactivar"
+SpiderballActivation_Hold = "Nova (mantenir)"
+SpiderballActivation_Hold_Tip = "Mantingues premut [Apuntar] per a activar"
+AutoClimb = "Auto Escalada"
+AutoClimb_Tip = "Escala petites plataformes [Endavant+Salt]"
+AutoMorph = "Morfosfera Automàtica"
+AutoMorph_Tip = "Activació automàtica [Mantenir A baix quan corres contra un túnel]"

```

should this not be 
```suggestion
AutoMorph_Tip = "Activació automàtica [Mantenir a baix quan corres contra un túnel]"
```
?

---

To be precise, to use the correct wording used in other places it should be "Avall"

```suggestion
AutoMorph_Tip = "Activació automàtica [Mantenir Avall quan corres contra un túnel]"
```